llvm/test/CodeGen/NVPTX/intrinsics.ll

```diff
@@ -140,6 +140,17 @@ define void @test_exit() {
   ret void
 }
 
+; CHECK-LABEL: test_globaltimer
+define i64 @test_globaltimer() {
+; CHECK: mov.u64         %r{{.*}}, %globaltimer;
+  %a = tail call i64 @llvm.nvvm.read.ptx.sreg.globaltimer()
```

Comment:
Thise need sm_30+. Right now the test runs with sm_30. LLVM does compile these intrinsics, but ptxas fails because the register is not available on sm_20.

The test needs to be updated to use a reasonably new GPU target. Probably sm_60 is the oldest one anybody still cares about.

Reply:
Okay, I can do that real quick.

Reply:
Done in https://github.com/llvm/llvm-project/commit/bb180856ec28efe305dc77ca4bb3db12d8932edf.